cmd/broker/retry/main.go

```diff
@@ -50,6 +50,9 @@ type envConfig struct {
 	// 3Mi. We also want to limit the memory usage from each subscription.
 	OutstandingBytesPerSub int `envconfig:"OUTSTANDING_BYTES_PER_SUB" default:"3000000"`
 
+	// MaxStaleDuration is the max duration of the handler pool without being synced.
+	MaxStaleDuration time.Duration `envconfig:"MAX_STALE_DURATION" default:"1m"`
```

Comment:
What's the reasoning behind this value or is it arbitrary?

Reply:
There is a maximum sync period of 15s. So I think the value should be greater than that. `1m` means we fail the probe if at least 4 sync loops have failed.

Reply:
That's right. I will add more comment.